app/errors.py

```diff
@@ -0,0 +1,19 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass

```

Comment:
Consider adding a descriptive message to the `OutdatedVaccineError` class to provide more context when the error is raised.